### PR TITLE
Rename package

### PR DESCRIPTION
**Pull request by jarrodmillman**
https://pypi.org/project/spin/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,72 +1,72 @@
 # Changelog
 
-## [v0.1](https://github.com/scientific-python/devpy/tree/v0.1)
+## [v0.1](https://github.com/scientific-python/spin/tree/v0.1)
```
Comment (stefanv) [inline CHANGELOG.md:3]: Should we leave this as-is, as only update upcoming releases?

File: pyproject.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [project]
-name = "devpy"
-version = "0.2rc1.dev0"
+name = "spin"
+version = "0.2b1"
```
Comment (stefanv) [inline pyproject.toml:3]: Just for my own education, what's the beta1 about?

**Review by stefanv — COMMENTED** [review]
Looks good, I will make a minor push to the PR with a README update as well.

File: pyproject.toml (hunk 2 above)
Comment (jarrodmillman) [reply pyproject.toml:3]: Whoops, not sure why I left that there...

**Review by stefanv — APPROVED** [review]

**Pull request merged**